test/integration/production/test/index.test.js

```diff
@@ -444,38 +444,43 @@ describe('Production Usage', () => {
     it('should add prefetch tags when Link prefetch prop is used', async () => {
       const browser = await webdriver(appPort, '/prefetch')
       await waitFor(2000)
-      const elements = await browser.elementsByCss('link[rel=prefetch]')
-
-      expect(elements.length).toBe(4)
-      await Promise.all(
-        elements.map(async element => {
-          const rel = await element.getAttribute('rel')
-          const as = await element.getAttribute('as')
-          expect(rel).toBe('prefetch')
-          expect(as).toBe('script')
-        })
+
+      const elements = await browser.elementsByCss(
+        'link[rel=prefetch],link[rel=preload]'
+      )
+
+      expect(elements.length).toBe(
+        global.browserName === 'internet explorer' ? 5 : 9
```

Comment:
Hmm, this doesn't seem right. It should be the current for everyone but Safari.

Reply:
This was from the `IntersectionObserver` not being available in `ie11` we can look into this test separately as it's back to Chrome only for now from https://github.com/zeit/next.js/pull/9825